AdminWebsite/AdminWebsite/AdminWebsite.csproj

```
@@ -31,7 +31,7 @@
     <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="BookingsApi.Client" Version="1.43.32" />
+    <PackageReference Include="BookingsApi.Client" Version="1.44.2-pullrequest0637-0013" />
```

Note to change this before you merge

---

yes thank you

---

This needs to be updated to use the sds feed. The pre-release packages include the branch name
```suggestion
    <PackageReference Include="BookingsApi.Client" Version="1.44.3-vih-9847---tl-ad0014" />
```